### ISSUE TITLE
Anno 1701 crashes in game with a DxvkMemoryAllocator error

### ISSUE THREAD

**Issue by WinterSnowfall** [doitsujin/dxvk#1410]
Anno 1701/Anno 1701: The Sunken Dragon (all settings maxed out, 1080p) crashes almost immediately after starting a mission with the following error:

```
err:   DxvkMemoryAllocator: Memory allocation failed
err:     Size:      4102400
err:     Alignment: 256
err:     Mem flags: 0x6
err:     Mem types: 0x681
err:   Heap 0: 326 MB allocated, 229 MB used, 349 MB allocated (driver), 8001 MB budget (driver), 8192 MB total
err:   Heap 1: 472 MB allocated, 465 MB used, 476 MB allocated (driver), 24030 MB budget (driver), 24030 MB total
terminate called after throwing an instance of 'dxvk::DxvkError'
```

### Software information
Anno 1701 + The Sunken Dragon Addon (GOG version).

### System information
- GPU: GeForce GTX 1070
- Driver: 440.48.2
- Wine version: 5.1 Staging
- DXVK version: v1.5.3-4-g08fc534

### Apitrace file(s)
Will provide one if needed.

### Log files
- d3d9.log: [Anno1701AddOn_d3d9.log](https://github.com/doitsujin/dxvk/files/4167516/Anno1701AddOn_d3d9.log)
- d3d11.log: N/A
- dxgi.log: N/A

**Comment by doitsujin** [comment]
`export WINE_LARGE_ADDRESS_AWARE=1` should fix this, see #1318.

**Comment by WinterSnowfall** [comment]
> Setting `WINE_LARGE_ADDRESS_AWARE=1` should fix this, see #1318.

... silly me, it's probably the first thing I should have tried. You're right, it seems stable now. Granted this does not happen with WineD3D, but there's [#1318](https://github.com/doitsujin/dxvk/issues/1318) to find a potential solution for the larger issue at hand.

Now that it works for more than 5 seconds, I have more bugs to file :).
